source/Engine.cpp

```diff
@@ -1179,6 +1226,15 @@ void Engine::Draw() const
 	draw[currentDrawBuffer].Draw();
 	batchDraw[currentDrawBuffer].Draw();
 
+	for(const Gunsight &gunsight : gunsights)
+	{
+		Point end1 = gunsight.start + gunsight.angle.Rotate(Point(gunsight.spread, -gunsight.range));
+		Point end2 = gunsight.start + gunsight.angle.Rotate(Point(-gunsight.spread, -gunsight.range));
+
+		LineShader::DrawGradient(gunsight.start * zoom, end1 * zoom, 1., gunsight.color, Color(0, 0, 0, 0));
+		LineShader::DrawGradient(gunsight.start * zoom, end2 * zoom, 1., gunsight.color, Color(0, 0, 0, 0));
```

Comment:
Move the second gradient color (currently zeroed) to interfaces, too.

Reply:
Well, that's the whole point of interface customizability. Even if it's going to stay as 0 alpha on vanilla, it doesn't mean we shouldn't provide a way to change it.